google-cloud-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -51,7 +51,7 @@
   private final double consistency;
   private int port;
```

Comment:
no need to have local field port, please remove it

Reply:
done